pandas/io/tests/test_clipboard.py

```diff
@@ -80,6 +82,14 @@ def test_round_trip_frame(self):
         for dt in self.data_types:
             self.check_round_trip_frame(dt)
 
+    def test_linesep(self):
+        pd.DataFrame().to_clipboard()
+        ret = pandas.util.clipboard.clipboard_get()
+        assert ret == '""'+os.linesep, "Wrong line separator"
```

Comment:
Well, i check that text data in the clipboard has intended separators (os.linesep). How can this be done with `assert_frame_equal`?